app/models/category.rb

```diff
@@ -0,0 +1,10 @@
+class Category < ApplicationRecord
+
+  #------- PLUGINS -------#
+  extend FriendlyId
+  friendly_id :name, use: :slugged
+
+  #------- VALIDATIONS -------#
+  validates :name, presence: true
+
```

Comment:
add relation that it can belong to another parent category and can have many sub categories. 
https://stackoverflow.com/a/43516714